.github/workflows/_control-plane.yml

```
@@ -0,0 +1,101 @@
+name: Control plane
```

Might be a tad more clear.

```suggestion
name: "Build artifacts: control plane"
```

---

I just followed the naming we have for the other workflows but I can update those too.